src/test/java/parser/ParserTest.java

```diff
@@ -0,0 +1,88 @@
+package parser;
+
+import duke.DukeException;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ParserTest {
+    @Test
+    public void convertTaskInput_deadlineCommand_success() throws DukeException {
```

Comment:
I like the adherence to the suggested structure "featureUnderTest_testScenario_expectedBehavior()" as per the coding standard for test cases.